.github/workflows/ci.yml

```diff
@@ -65,6 +65,20 @@ jobs:
       - name: test:feature-flags
         run: yarn run ember try:one ember-data-packages-canary --- ember test --query enableproxy
 
+  test-performance-app:
+    name: performance-app
+    needs: [lint]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x' # min node version
+      - name: yarn install
+        run: cd m3-perf-testing-app && yarn --frozen-lockfile --install
+      - name: test:performance-app
+        run: cd m3-perf-testing-app && yarn test 
```

Comment:
FWIW, you don't have to `cd ... && other-thing` like this, you can tell GH actions to use whatever working dir you want:

```suggestion
  test-performance-app:
    name: performance-app
    needs: [lint]
    runs-on: ubuntu-latest
    
    # run all of the commands below inside the perf test app
    defaults:
      run:
        working-directory: m3-perf-testing-app

    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: '12.x' # min node version
      - name: yarn install
        run: yarn --frozen-lockfile --install
      - name: test:performance-app
        run: yarn test 
```

Alternatively you could specify the working directory for each `run:` command like this:

```js
  test-performance-app:
    name: performance-app
    needs: [lint]
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: '12.x' # min node version
      - name: yarn install
        run: yarn --frozen-lockfile --install
        working-directory: m3-perf-testing-app
      - name: test:performance-app
        run: yarn test 
        working-directory: m3-perf-testing-app
```

Reply:
Some docs on `defaults.run` here: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_iddefaults

Reply:
Oh neat thanks!